components/keyboard_shortcuts/keyboard_shortcuts_modal/keyboard_shortcuts_modal.tsx

```diff
@@ -54,28 +54,25 @@ const modalMessages = defineMessages({
     },
 });
 
-const KeyboardShortcutsModal = (): JSX.Element => {
-    const [show, setShow] = useState<boolean>(false);
+interface Props {
+    onExited: () => void;
+}
 
-    useEffect(() => {
-        //toggles the state of shortcut dialog
-        const handleToggle = (): void => setShow(!show);
-        ModalStore.addModalListener(Constants.ActionTypes.TOGGLE_SHORTCUTS_MODAL, handleToggle);
-        return () => {
-            ModalStore.removeModalListener(Constants.ActionTypes.TOGGLE_SHORTCUTS_MODAL, handleToggle);
-        };
-    }, []);
+const KeyboardShortcutsModal: FC<Props> = (props: Props) => {
+    const [show, setShow] = useState(true);
 
-    const handleHide = (): void => setShow(false);
     const {formatMessage} = useIntl();
+
+    const handleHide = (): void => setShow(false);
```

Comment:
This should be wrapped in a `useCallback` to prevent the `Modal` from re-rendering whenever this does

Reply:
wrapped it in callback hook